npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -77,7 +76,9 @@ def self.npm_version_numeric_npm8_or_higher(lockfile)
 
         lockfile_version = lockfile_version_str.to_i
 
-        return NPM_V9 if lockfile_version == 3
+        # Using npm 8 as the default for lockfile_version > 2.
+        # Update needed to support npm 9+ based on lockfile version.
+        return NPM_V8 if lockfile_version >= 2
```

Comment:
this means we'll use npmv8 for lockfilev3 too?

Reply:
Yes, this is what we were using before.

Reply:
We just replaced defaults from npm 8 to npm 8